lib/libc/rules.mk

```diff
@@ -10,6 +10,7 @@ MODULE_SRCS += \
 	$(LOCAL_DIR)/abort.c \
 	$(LOCAL_DIR)/atexit.c \
 	$(LOCAL_DIR)/atoi.c \
+	$(LOCAL_DIR)/atof.c \
```

Comment:
You'll want to move this to the MODULE_FLOAT_SRCS section so that appropriate architecture will use floating point compiler switches.

Reply:
ah, right.